modules/client-configuration/pages/autoinst-reinstall.adoc

```diff
@@ -0,0 +1,14 @@
+[[reinstalling-registered-systems]]
+= Reinstalling Registered Systems
+
+To reinstall in place an already registered client, you first have to define an autoinstallation distribution and an autoinstallation profile. For details, see xref:client-configuration:autoinst-xxxxx.adoc[] and xref:client-configuration:autoinst-xxxxx.adoc[].
+
+Once this is done, navigate to menu:Systems[Systems List], select the client to reinstall, and select menu:Provisioning[Autoinstallation > Schedule] subtab.
+
+At this point, select the autoinstallation profile that you prepared, select a proxy if needed, and press btn:[Schedule Autoinstallation and Finish]. If your client is a traditional client, type [command]``mgr_check`` on that client.
+
+You may follow the progress of the installation in menu:Provisioning[Autoinstallation > Session Status], or simply on the client's screen. You will see the client reboot, and automatically select in the boot menu a new choice called [guimenu]``reinstall-system``.
+
+image::autoinstall_reinstall.png[scaledwidth=80%]
+
+This being a purely local mechanism, there is no need for PXE booting capabilities on the client.
```

Comment:
```suggestion
Because PXE booting happens entirely on the local system, there is no need for PXE booting capabilities on the client.
```

This sentence is stuck all alone under the image, can we attach it to a para?

Reply:
Your suggested change, no, no, no! The point is, PXE booting is not involved at all here! :smile_cat:

Is there a way to attach the image to the procedure above ? It's where it belongs to.